addins/Cake.NDepend.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.NDepend
 NuGet: Cake.NDepend
-Prerelease: false
 Assemblies:
 - "/**/Cake.NDepend.dll"
-Repository: https://github.com/cake-contrib/cake-ndepend
+Repository: https://github.com/joaoasrosa/cake-ndepend
```

Comment:
@Jericho is this a problem in the NuGet metadata as well?  If so, I think we can get this one fixed.

Reply:
If you fix the nuspec, then this problem goes away. Another reason why AddinDiscoverer was unable to figure out that this project has been moved to cake-contrib is because the name of the nuget package (i.e. Cake.NDepend) does not match the name of the github repo (i.e.: cake-ndepend). Notice in one case the '-' is used between `Cake` and `NDepend` and in the other case '.' is used.